chia/rpc/full_node_rpc_client.py

```diff
@@ -200,6 +200,16 @@ async def get_block_records(self, start: int, end: int) -> List:
         # TODO: return block records
         return response["block_records"]
 
+    async def get_block_spends(self, header_hash: bytes32) -> Optional[List[CoinSpend]]:
+        try:
+            response = await self.fetch("get_block_spends", {"header_hash": header_hash.hex()})
+            block_spends = []
+            for block_spend in response["block_spends"]:
+                block_spends.append(CoinSpend.from_json_dict(block_spend))
+            return block_spends
+        except Exception:
```

Comment:
how does the caller know what went wrong here? we don't log any exceptions, just catch them and return None

Reply:
Its what every other api call does.

Reply:
I don't think we have to catch anything. The RPC server will catch it and return the error string to the caller. In this case we don't know what failed